experiment_dirs

```diff
@@ -60,7 +60,7 @@ for group in $*; do
     esac
 done
 
-EXCLUSIONS=( )
+EXCLUSIONS=(/g/data/ik11/outputs/access-om2-01/basal_melt_outputs)
```

Comment:
should this be 
```
/g/data/ik11/outputs/access-om2-01/basal_melt_outputs/accessom2-GPC001
```
?

Reply:
I tried that, but what is in the list is `/g/data/ik11/outputs/access-om2-01/basal_melt_outputs/`, not `/g/data/ik11/outputs/access-om2-01/basal_melt_outputs/accessom2-GPC001`.